_docs/style/text-conventions/labels.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Labels
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  A label is a graphical element that displays text on control panel page. Use a label to identify sections on a page and navigation menu items.
```

Comment:
I was under the impression that each menu item is identified by a label? Maybe I'm wrong about that? We should chat about this...

Reply:
Ok, I did a bit more research on 'label' and found this: "A label is a graphical control element which displays text on a form. It is usually a static control; having no interactivity."

...which means, that you are correct, a menu item is NOT a label, since it is interactive. Since labels are static, we're left with content labels and menu groups.